### DISCUSSION TITLE
Problem with FieldItemList in latest release

### DISCUSSION THREAD

**Discussion by DuttonMa** [mglaman/phpstan-drupal#510] [Q&A] [answered]
When running drupal-check with the latest version of phpstan (1.1.28) yesterday I received the following error:

Line   ../vendor/mglaman/phpstan-drupal/src/Drupal/../../stubs/Drupal/Core/F  
         ield/FieldItemList.stub                                                
 ------ ----------------------------------------------------------------------- 
  7      Class Drupal\Core\Field\FieldItemList extends unknown class            
         Drupal\Core\TypedData\Plugin\DataType\ItemList.                        
  7      Class Drupal\Core\Field\FieldItemList implements generic interface     
         Drupal\Core\Field\FieldItemListInterface but does not specify its      
         types: T                                                               
  12     Method Drupal\Core\Field\FieldItemList::createItem() has parameter     
         $offset with no type specified.                                        
  12     Method Drupal\Core\Field\FieldItemList::createItem() has parameter     
         $value with no type specified.                                         
 ------ ----------------------------------------------------------------------- 
 [ERROR] Found 4 errors      

So I think that some changes are needed to the FieldItemList.stub ?

**Comment by mglaman** [comment, marked as answer]
This is the same as https://github.com/mglaman/phpstan-drupal/issues/509

It has been fixed with https://github.com/mglaman/phpstan-drupal/releases/tag/1.1.29
